packages/suite-native-settings/src/components/SettingsSection.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactNode } from 'react';
+
+import { Box, Text, VStack } from '@suite-native/atoms';
+
+type SettingsSectionProps = {
+    title: string;
+    children: ReactNode;
+    rightIcon?: ReactNode;
```

Comment:
I would prefer go with 
```tsx
rightIconName: IconName
onRightIconPress: () => void
```

Passing whole component is not necessary because we don't need to customize it that much

Reply:
[8c23b9f](https://github.com/trezor/trezor-suite/pull/5617/commits/8c23b9f9cafd67684e4e1e0e3c5c399d53c6bc34)